package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "service-name",
+  "name": "external-to-geojson",
   "version": "1.0.0",
-  "description": "This is template for map colonies typescript service",
+  "description": "convert external format to geoJson format",
   "main": "./src/index.ts",
   "scripts": {
     "test:unit": "jest --config=./tests/configurations/unit/jest.config.js",
```

Comment:
your deletion of  `package-lock.json` is wrong, a new `package-lock.json` should be introduced